backend/lib/spree/backend_configuration.rb

```diff
@@ -6,6 +6,22 @@ module Spree
   class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: I18n.default_locale
 
+    # @!attribute [rw] themes
+    #   @return [Hash] A hash containing the themes that are available for the admin panel
+    preference :themes, :hash, default: {
+      classic: 'spree/backend/all',
+    }
+
+    FALLBACK_THEME = :classic
+
+    # @!attribute [rw] theme
+    #   @return [String] Default admin light theme name
```

Comment:
The final work in this PR is agnostic to light/dark concepts. What do you think if we remove references to it (also in the specs below)?